adal/devApps/XFormsApp/SecondPage.cs

```diff
@@ -63,9 +63,9 @@ public SecondPage()
                 Text = "Conditional Access"
             };
 
-            var clearButton = new Button
```

Comment:
In the MSAL Xamarin Dev app, we have all three options (`ClearAdalCache`, `ClearMsalCache`, `ClearAllCache`)...but in adal, just `ClearAllCache` ...making a note of this here as something to add later for testing.